### PR TITLE
Make a Python package, prepare for PyPI release

### PR DESCRIPTION
**Pull request by gdubicki**

### CODE REVIEW

**Comment by jakunow** [comment]
@gdubicki I've change base branch from **main** to **develop** - want to test those changes first.

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import codecs
+
+from setuptools import setup, find_packages
+
+with codecs.open("README.md", encoding="utf-8") as f:
+    readme = f.read()
+
+setup(
+    name="cloudimized",
+    version="1.0.0",
```
Comment (jakunow) [inline setup.py:10]: Current version is 1.1.0

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        "Intended Audience :: Information Technology",
+        "Intended Audience :: System Administrators",
+        "License :: OSI Approved :: MIT License",
+        "Operating System :: POSIX :: Linux",
+        "Operating System :: MacOS",
+    ],
+    packages=find_packages(),
+    package_data={"": ["LICENSE", "*.md", "config-example.yml"]},
+    include_package_data=True,
+    install_requires=[
```
Comment (jakunow) [inline setup.py:31]: With recent new feature requirements list needs to be updated.

**Review by jakunow — COMMENTED** [review]
@gdubicki after adding new feature there are some conflicts now, some references in new code will also need to be adjusted.

Is there a way we can test this PyPI code in develop, before pushing it to main? I see references in code that point to 'main' branch

File: setup.py (hunk 2 above)
Comment (gdubicki) [reply setup.py:31]: Updated.

File: setup.py (hunk 1 above)
Comment (gdubicki) [reply setup.py]: Thanks, fixed.

**Comment by gdubicki** [comment]
Can you please make these 2 tests pass, @jakunow? I don't seem to understand their logic (at this hour ;)) well enough to do it myself and for you it will probably be trivial. :)

**Comment by jakunow** [comment]
@gdubicki Issue with tests fixed. Test where failing when running unittest in discover mode, but running individual tests were passing. Issue was that I've disabled logging on a module level and that setting was carried to next file - see [ff4579b30ca90463dfd1158fd472150c363e6537 ](https://github.com/egnyte/cloudimized/pull/6/commits/ff4579b30ca90463dfd1158fd472150c363e6537) for details

**Pull request merged**